test/lint/lint-locale-dependence.sh

```diff
@@ -45,7 +45,6 @@ KNOWN_VIOLATIONS=(
     "src/test/dbwrapper_tests.cpp:.*snprintf"
     "src/test/fuzz/locale.cpp"
     "src/test/fuzz/string.cpp"
-    "src/torcontrol.cpp:.*strtol"
```

Comment:
if this was the last occurence as you say, could the TODO in line 40/41 be removed?

Reply:
thx, fixed.